docs/rules/lines-around-comment.md

```diff
@@ -172,6 +172,55 @@ function foo(){
 }
 ```
 
+### allowClassStart
+
+Examples of **correct** code for this rule with the `{ "beforeLineComment": true, "allowClassStart": true }` option:
```

Comment:
Can you add a few incorrect code examples too. For example, `{ "beforeLineComment": true, "allowClassStart": false }`, or `{ "beforeLineComment": false, "allowClassStart": true }`